custom_dict.txt

```diff
@@ -8,6 +8,7 @@
 [0-9]*(a|b|c)
 [0-9]*(am|pm|AM|PM)
 [0-9]*(GB|MB|KB|Mi|m|m?s|k)
+[A-Za-z0-9]{11}
```

Comment:
What is this catching?

Reply:
I had a comment on this but it was messing up the actual check, I'll double check to see if I can add it, but this was for YouTube video IDs. We were adding them one-by-one, which I suppose we can still do

Reply:
I actually went ahead and changed this to just add the YouTube ID like we have been. The more I thought about it, the right route is to update the spellchecker to skip certain shortcodes